<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaNotaAddDlg</class>
 <widget class="QDialog" name="PrimaNotaAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::DaySection</enum>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>10</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descrizione:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Entrate</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Cassa:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Banca:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="entBancaLineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="entCassaLineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Uscite</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Cassa:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="uscCassaLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Banca:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="uscBancaLineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Chiudi</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Cancella</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Salva</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>entCassaLineEdit</tabstop>
  <tabstop>entBancaLineEdit</tabstop>
  <tabstop>uscCassaLineEdit</tabstop>
  <tabstop>uscBancaLineEdit</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>comboBox</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>entCassaLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>entBancaLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>uscCassaLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>uscBancaLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>PrimaNotaAddDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>PrimaNotaAddDlg</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entCassaLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PrimaNotaAddDlg</receiver>
   <slot>updateLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entBancaLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PrimaNotaAddDlg</receiver>
   <slot>updateLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uscCassaLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PrimaNotaAddDlg</receiver>
   <slot>updateLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uscBancaLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PrimaNotaAddDlg</receiver>
   <slot>updateLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>updateLineEdit()</slot>
 </slots>
</ui>
